### ISSUE TITLE
[Keyboard Navigation – Make Code - Publish Project] Tab items are accessible via tab key present under 'Share Project' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3603]
**User Experience:**
Keyboard dependent users will be impacted if controls are not accessible through their standard keys. Users expect that radio buttons should be accessible through arrow keys, and if they not, it will become quite confusing for them during navigation.

**Note:** User credentials should NOT be included in the bug. 

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate to "Publish project" under  "Share Project" dialog and select it.
7. Navigate to all the control appearing like QR Code, Embed(Code, Editor and Simulator).
8. Verify that Tab items(Code, Editor & Simulator) are accessible via tab key.

**Actual Result:**
Tab items (Code, Editor & Simulator) are accessible via tab key present under 'Share Project' pop-up window.

**Expected Result:**
Tab items (Code, Editor & Simulator) should not accessible via tab key present under 'Share Project' pop-up window. It should only accessible via arrow keys.

**Note:**
1. Same issue exists on Firefox & Chrome.
2. Same issue exists on menu items present under 'Help & More' icon controls present on header section.

**WCAG Reference:**
[WCAG 2.1.1 - Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.1.1, #Win10-Edge(Chromium)

[WCAG2.1.1_Publish Project.zip](https://github.com/microsoft/pxt-microbit/files/5499365/WCAG2.1.1_Publish.Project.zip)

**Comment by jwunderl** [comment]
embed content fixed, should probably add keyboard tab for qr code button
